electron/main.ts

```diff
@@ -80,6 +81,13 @@ const tsStore = new Store({
   name: 'timestamp'
 })
 
+// Trigger the autoUpdater every X minutes
+const interval = setInterval(() => checkUpdates(), (GlobalConfig.get().config?.checkUpdatesInterval || 10) * 60000) // Converts minutes to milliseconds
+
+if (GlobalConfig.get().config.enableUpdates === false) {
```

Comment:
@dragonDScript I think this line is the problem, just remove the `=== false`its unnecessary since if nothing it will return false anyway.

Reply:
It should be true by default, so if it's undefined JS will interpret that it's false

Reply:
So then do: 
`GlobalConfig.get().config.enableUpdates === false || true`

Reply:
Also, I dont know if updates should be forced, we have a culture in Linux that the user chooses to upgrade or not, some things I upgrade only if it looks safe, sometimes updates breaks stuff

Reply:
so keep it as false by default the autoupdate, leave only the "checkUpdates" as true by default

Reply:
We should ask our users in the Discord. I'll leave the default as false by default by now